eZ Platform/APIs/Languages/README.md

```diff
@@ -0,0 +1,70 @@
+# v2: Updated API Language behaviour
+
+## Objective
+
+Simplify usage of the product for developers, avoid common mistakes when dealing with multi language installs,
+and make sure to cover needs for future UI needs.
+
+### Use Cases
+The identified new behaviour:
+- As a Developer, I want to get API entities in prioritised language of the given SiteAccess,
+  so I don't need to specify language in API Service calls
+- As a Developer, I want to get API entities in one Translation at a time,
+  so I don't need to specify language or use helpers when using the API entities
+
+##### Related use cases
+- As an Editor, in UI I want to be able to select languages of API entities to view and edit translation independently
+  of the current SiteAccess, _so I don't need to deal with several variants of backend to do that_
```

Comment:
Hi @supriyabhargava I can try to expand, it's written in a way that hopefully should be understandable for everyone experienced with legacy, but with additional context maybe you have suggestion on how I can rather word this.

The missing context: In legacy, admin (backend) was a siteaccess just link any fronted siteaccess. And language for the interface was set in siteaccess settings. This meant that if you had translation teams, or otherwise different people that wanted admin in different languages you would have to create several siteaccesses, like you would for frontend.

The downside of this was 1. the system does not know what all these siteaccesses are for, are they for a custom channel, are they for a given language for frontend, or are they for a relevant alternative language of admin? 2. It means editors would need to use different url/host for the different languages, there was no drop down to select language for interface.

Additionally as is the case on the requirement we comment on here intention is to clarify that we still want to allow editor to select language to view/edit the translations of content _(I'm using the word entity in the requirement to also cover data like sections, object states, content types and repository languages, who are all aiming to be translatable as well)_, in addition to UI operations to manage translations.

---

So the purpose of these Editorial requirements _(one for entity translations and one for interface language)_ is to clarify where we intend to go in regards to the interface, namely letting the backend user select interface language on for instance login screen making it integrated part of the experience. This is defined here as it will have effect for changes we do on language handling in the UI system as part of v2 changes _(some of the changes might also make it into 1.x as @rolandbenedetti mentions, but intention is to leave no room for doubt on this)_.